bin/dao/users_dao.dart

```diff
@@ -1,15 +1,35 @@
+import 'package:password_dart/password_dart.dart';
+import 'package:uuid/uuid.dart';
+
 import '../database/db_configuration.dart';
 import '../models/user_model.dart';
 import 'dao.dart';
 
 class UserDAO implements DAO<UserModel> {
   final DBConfiguration _dbConfiguration;
-  UserDAO(this._dbConfiguration);
+  final Uuid uuid;
+
+  UserDAO(this._dbConfiguration, this.uuid);
 
   @override
-  Future<bool> create(UserModel value) {
-    // TODO: implement create
-    throw UnimplementedError();
+  Future<bool> create(UserModel value) async {
+    final DateTime now = DateTime.now().toUtc();
+    final String password = value.password ?? "";
```

Comment:
Neste ponto , proponho que ao invés de tratar um *password* nulo como vazio, verificar se o *password* for nulo e já retornar um *badRequest*.